client/src/curriculum/scrim.tsx

```diff
@@ -0,0 +1,81 @@
+import { useState, useRef } from "react";
+
+import { ReactComponent as EnterFullscreen } from "../../public/assets/curriculum/enter-fullscreen.svg";
+
+import "./scrim.scss";
+
+export function PartnerIframe({
```

Comment:
Why not call it `Scrim` or `ScrimIframe`, given that the module is called `scrim.tsx`? Or maybe rename this module to `partner-iframe.tsx` instead.

```suggestion
export function ScrimIframe({
```